collector/database/classes/collector.php

```diff
@@ -143,7 +143,7 @@ public function get_metrics_aggregated($metricnames = null, $since = false, int
         } else {
             $metricselect = '';
             foreach ($params as $param) {
-                $metricselect .= "AVG(CASE name WHEN '$param' THEN" . $DB->sql_cast_char2int('value', true) . "END) $param,";
+                $metricselect .= "AVG(CASE name WHEN '$param' THEN" . $DB->sql_cast_char2int('value', true) . "END) \"$param\",";
```

Comment:
This SQL looks odd to me. Does it actually work? It looks like it's missing an "AS".

Reply:
It is working without but following your suggestion, I added it to the SQL. Thanks.

Reply:
I think it would fail when using Oracle.

While we're playing with it, it's really odd that it uses `get_in_or_equal` because the `$clause` part is never used. Would we be able to change it to do something like `foreach $metricnames as $metricname {` and get rid of the `get_in_or_equal`. It looks like it's a copy+paste error.

Reply:
Agree with the SQL changes. 
Regarding the `get_in_or_equal()` the $clause is actually used  [here](https://github.com/catalyst/moodle-tool_cloudmetrics/blob/MOODLE_35_STABLE/collector/database/classes/collector.php#L92). Would you still recommend removing it?

Reply:
Oops, I'm blind :) Let's leave it then.